heroku/cli
# refactor(pg-v5): Move command pg:maintenance:window to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bCiEWYA0/view

This converts `pg:maintenance:window` to oclif and updates its associated tests.

### Testing

1) Pull down this branch
2) Run `./bin/run pg:maintenance:window --app APP DATE` with an app that has postgres where DATE is in the format of "Day HH:MM" (e.g., "Wednesday 14:00")
3) Verify the command succeeds in setting the maintenance window -- `heroku pg:maintenance --app APP` to see the value set
4) Compare with prod
